src/Java.Interop.Tools.Cecil/Java.Interop.Tools.Cecil/DirectoryAssemblyResolver.cs

```diff
@@ -187,7 +196,7 @@ public AssemblyDefinition Resolve (AssemblyNameReference reference, ReaderParame
 			AssemblyDefinition candidate = null;
 			foreach (var dir in SearchDirectories) {
 				if ((assemblyFile = SearchDirectory (name, dir)) != null) {
-					var loaded = Load (assemblyFile);
+					var loaded = Load (assemblyFile, parameters);
```

Comment:
no, there's already a version with `ReaderParameters` on line 152.
thanks for the hint about default parameter values, I'll change that.